draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6404,16 +6401,21 @@ Expect: 100-continue
    parameters that are applicable to that range.
 </t>
 <t>
-   Each media-range might be followed by zero or more applicable media type
-   parameters (e.g., <x:ref>charset</x:ref>), an optional "q" parameter for
-   indicating a relative weight (<xref target="quality.values"/>), and then zero or more extension
-   parameters. The "q" parameter is necessary if any extensions (accept-ext) are present,
-   since it acts as a separator between the two parameter sets.
+   Each media-range might be followed by optional applicable media type
+   parameters (e.g., <x:ref>charset</x:ref>), followed by an optional "q"
+   parameter for indicating a relative weight (<xref target="quality.values"/>).
+</t>
+<t>
+   This specification does not define "accept parameters" following the 
+   special "q" parameter, as was done in earlier editions. Senders using
```

Comment:
Editions doesn't seem quite right - what language do we use in other places? Revisions?

Reply:
More like
> Previous specifications of Accept allowed additional extension parameters to appear after the weight parameter. The accept extension grammar has been removed because they had a complicated definition, were not being used in practice, and more easily deployed through new header fields.